### ISSUE TITLE
Add instructions for installing on Debian

### ISSUE THREAD

**Issue by silentninja** [mathesar-foundation/mathesar#3165]
We need to add instructions for installing the Debian package to our installation documentation. [This](https://wiki.mathesar.org/en/projects/installation-improvements-plan-0_1_4#install-on-debian) outline for the documentation, the instructions should be based on the outline

**Comment by seancolsen** [comment]
This was previously blocked by #3164 but now appears to be ready as far as I can tell.

**Comment by kgodey** [comment]
@seancolsen I'm not sure this is ready, IIRC in the last discussion I had with @mathemancer about this, we realized we needed to clarify requirements and implementation specs. @mathemancer, please confirm.

**Comment by mathemancer** [comment]
@kgodey The last place I remember leaving this was that we're not planning to offer the Debian package for 0.1.4. So, we don't need to worry about this much at the moment. I'd argue that this should not be a separate issue from the preparation of the Debian package for publication, and we should probably close this and move tracking publication of the   Debian package to Basecamp. What do you think?

**Comment by kgodey** [comment]
@mathemancer That all sounds great to me, I'll leave it to you to make the changes.

**Comment by kgodey** [comment]
Closing as discussed above.
